src/App.css

```diff
@@ -48,40 +74,40 @@ footer {
   width: 100%;
   height: auto;
   margin: 0 auto;
-  padding: 30px 0px;
-  background-color: #2f97f2;
+  padding: 24px 0px;
 }
 
 footer .footerin {
-  width: 98%;
+  /* width: 98%;
```

Comment:
use double star for opening comment `/** like this */` ; this is for all css/scss/less files here.
if you are sprinkling comments add a `TODO(<github handle): <reason for the comment`;
Once necessary step has been taken, remove that comment